### PR TITLE
Updated readme.md with API reference

### PR DESCRIPTION
**Pull request by petermorgandev**
I started the API documentation for #29.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +35,27 @@ npm install
...
-
 # Contributing
 
-We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request. 
\ No newline at end of file
+We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request.
+
+# API Reference
+
+`https://github-graphs.com/api/friends/<username>`
```
Comment (jrtechs) [inline README.md]: We should make it clear that this is a GET request and provide a small sample of what it returns.

File: README.md [hunk 2]
```diff
@@ -37,7 +35,27 @@ npm install
...
+
+- `https://api.github.com/users/<username>`
+- `https://api.github.com/users/<username>/followers`
+- `https://api.github.com/users/<username>/following`
+- `https://api.github.com/users/<username>/orgs`
```
Comment (jrtechs) [inline README.md]: Doesn't actually get called with the friends request, this is only called when we try to render the users profile

File: README.md [hunk 3]
```diff
@@ -37,7 +35,27 @@ npm install
...
-We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request. 
\ No newline at end of file
+We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request.
+
+# API Reference
+
+`https://github-graphs.com/api/friends/<username>`
+
+- `https://api.github.com/users/<username>`
+- `https://api.github.com/users/<username>/followers`
```
Comment (jrtechs) [inline README.md]: We could possibly link to the GH API here.

File: README.md [hunk 4]
```diff
@@ -37,7 +35,27 @@ npm install
...
+
+`https://github-graphs.com/api/repositories/<username>`
+
+- `https://api.github.com/users/<username>/repos`
+
+`https://github-graphs.com/api/org/users/<username>`
+
+- `https://api.github.com/orgs/<username>/members`
+
+`https://github-graphs.com/api/org/repositories/<username>`
```
Comment (jrtechs) [inline README.md]: We should provide a high-level explanation for each API endpoint, like "returns all repositories that are owned by a GH organization"

**Review by jrtechs — CHANGES_REQUESTED** [review]
@PeterMorganGH nice start! I think that we should add some more details to make it easier for other people to jump into this project

File: README.md (hunk 2 above)
Comment (petermorgandev) [reply README.md]: I removed the `/orgs` link is in https://github.com/PeterMorganGH/github-graphs/commit/490ff1709dd6f06c4052c6c321d28d96ff426e5f.

File: README.md (hunk 3 above)
Comment (petermorgandev) [reply README.md]: Done

**Comment by petermorgandev** [comment]
I updated `readme.md` in this commit: https://github.com/PeterMorganGH/github-graphs/commit/71bfeb476672ec89db62a670ccde8edb1ddb2e5a. Let me know if you would like additional changes.

**Review by jrtechs — APPROVED** [review]
Looks great @PeterMorganGH :+1:

**Pull request merged**